common/src/main/java/cz/incad/kramerius/audio/servlets/ServletAudioHttpRequestForwarder.java

```diff
@@ -120,8 +120,11 @@ private void forwardData(InputStream input, ServletOutputStream output) {
                     && (ex.getCause().getMessage().equals(CONNECTION_RESET) || ex.getCause().getMessage().equals(BROKEN_PIPE))) {
                 LOGGER.warning("Connection reset probably by client (or by repository)");
             } else {
-                LOGGER.log(Level.SEVERE, null, ex);
-            }
+                if ("ClientAbortException".equals(ex.getClass().getSimpleName())) {
+                    LOGGER.info("Bad sound unit image");
+                } else {
+                    LOGGER.log(Level.SEVERE, ex.getMessage(), ex);
+                }            }
```

Comment:
Špatně zalomený kód
Nerozumím te hlášce, skutečně je špatný soundunit image? ClientAbortException nastává, když http klient přeruší spojení. Když si to otevřeš ve Firefoxu, tak uvidíš, které ta spojení jsou předčasně zavřena (dělá to například prohlžečka obrázků).

Reply:
Navíc kód je  v servletu na Audio, takže to musí způsobovat request na získání audia...

Ale zase asi nemá cenu s tím trávit moc času, za to ten problém nestojí. Uděláme to "tiché" (bez logování), jako v případě prohlížečky.